src/main/java/mate/academy/internetshop/controllers/ProductAddToCartController.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.internetshop.controllers;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.academy.internetshop.lib.Injector;
+import mate.academy.internetshop.service.ProductService;
+import mate.academy.internetshop.service.ShoppingCartService;
+
+@WebServlet("/products/add")
```

Comment:
```suggestion
@WebServlet("/products/addToCart")
```

Reply:
When I change it to addToCart it gives me 500 error

Reply:
Done